spec/features/publish_verification_results_and_version_spec.rb

```diff
@@ -49,22 +49,4 @@
     expect(last_response.status).to be 200
     expect(JSON.parse(subject.body)).to include JSON.parse(verification_content)
   end
```

Comment:
Have moved this spec out of the feature directory, into the integration test directory.